scripts/collect_steps.py

```diff
@@ -16,15 +17,22 @@
     "--features=enable-benches debug-trace step-trace",
     "--",
     "--num-multi-bits",
-    "1",
+    "3",
 ]
 QUERY_SIZE = 10
 PER_USER_CAP = [1, 3]
 ATTRIBUTION_WINDOW = [0, 86400]
-BREAKDOWN_KEYS = [1, 33]
+BREAKDOWN_KEYS = [8, 33]
```

Comment:
is there a difference between 1 and 8?

Reply:
There are steps that are dynamically generated based on the number of bits used to represent breakdown keys, and I realized that the coverage wasn't enough with "1". "8" is enough to cover all cases for 1-32 (specialized aggregation protocol). >=33 uses a generic protocol so one is enough to cover all.

Reply:
gotcha. I would suggest to maybe follow up with a comment that explains it inside the code as well. Does not have to be in this PR

Reply:
added comments